docs/environment-variables.md

```diff
@@ -0,0 +1,44 @@
+# Environment Variables
+
+For the `main` container of vertex pods, environment variable `NUMAFLOW_DEBUG` can be set to `"true"` for [debugging](./debugging.md).
```

Comment:
should we callout explicitly that the `true` should be in `"` ?

Reply:
Removed the ", since it's been called out in the next linked page.